controller/index.js

```diff
@@ -145,47 +143,22 @@ var Controller = function (agol, BaseController) {
     var options = req.query
     var id = params.id
     var item = params.item
-    var key = params.key
 
-    agol.find(id, function (err, data) {
-      if (err) {
-        return callback(err, null)
-      }
+    // Get the item
+    if (!parseInt(options.layer, 0)) {
```

Comment:
`parseInt` should have a radix of 10 (not 0) assuming decimal ([ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt))